Wrapping/Python/Python.i

```diff
@@ -653,6 +653,55 @@
 
 }
 
+
+%extend itk::simple::Transform {
+   %pythoncode %{
+
+        def __copy__(self):
+          """Create a SimpleITK shallow copy, where the internal image is shared with a copy on write implementation."""
```

Comment:
I believe this should say "internal transform"

Reply:
done